cmake/developer_package/IEDevScriptsConfig.cmake

```diff
@@ -50,7 +50,10 @@ endif()
 #
 
 function(set_temp_directory temp_variable source_tree_dir)
-    if (DEFINED ENV{DL_SDK_TEMP} AND NOT $ENV{DL_SDK_TEMP} STREQUAL "")
+    if(DEFINED OV_TEMP)
```

Comment:
why `OV_TEMP`? I suppose we should check `DEFINED ENV{DL_SDK_TEMP}`

Reply:
We support both variant. `OV_TEMP` is a cmake option